<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GOptimizingWidget</class>
 <widget class="QWidget" name="GOptimizingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GOptimizingWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="pBoxVariables">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Variables</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Algorithms from NLopt</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QComboBox" name="pAlgoComboBox"/>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pAlgoDescriptionEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pBoxAlgo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="pAlgoStackSettings">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pWaitingTimeBox">
       <property name="title">
        <string>Waiting time after iteration [s]</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pButtonRun">
       <property name="text">
        <string>Run</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="pBoxParamOptimize">
     <property name="title">
      <string>Param to Optimize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pBoxHistory">
     <property name="title">
      <string>History</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="pButtonClearHistory">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="pTableHistory">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Fitness</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pButtonClearHistory</sender>
   <signal>clicked()</signal>
   <receiver>pTableHistory</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
